kitsune/sumo/static/sumo/js/device-migration-wizard.js

```diff
@@ -123,21 +125,133 @@ class ConfigureStep extends BaseFormStep {
     return `
       <template>
         <div>
-          <p>This is where the sync step will go.</p>
-          <p id="sync-status"></p>
+          <p id="header">
+            <img class="icon" src="${infoImageURL}" aria-hidden="true"></img>
+            <span>${gettext("You are now logged in to Firefox Accounts")}</span>
+            <a href="#">${gettext("Forgot password?")}</a>
```

Comment:
@mikeconley Do you think we should report clicks on this to GA? If so, I guess we could use:
- `data-event-category="device-migration-wizard"`
- `data-event-action="click"`
- `data-event-label="forgot-password"`

Reply:
Good idea. I'll add that.